src/Services/GTT/shared/GTT.Application/Response/BaseResponseModel.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Net;
+
+namespace GTT.Application.Response
+{
+    public class BaseResponseModel
```

Comment:
each api should has it own response type, change the name BaseResponseModel to something more specific

Reply:
The purpose of creating this model is to form a common data return standard, it includes status code, message, and data that change according to their intended use.